pandas/core/generic.py

```diff
@@ -2120,11 +2122,59 @@ def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
 
         >>> engine.execute("SELECT * FROM integers").fetchall()
         [(1,), (None,), (2,)]
+
+        Insertion method:
+
+        .. versionadded:: 0.24.0
+
+        The parameter ``method`` controls the SQL insertion clause used.
+        Possible values are:
+
+         - `'default'`: Uses standard SQL `INSERT` clause
+         - `'multi'`: Pass multiple values in a single `INSERT` clause.
+            It uses a **special** SQL syntax not supported by all backends.
```

Comment:
The 'It' should be indented equally to the `` `'multi'` `` of the line above (now it is one space more I think)